packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -200,14 +200,16 @@ const InputContainer = createComponentSlot("div")<FieldContainerClassKey, OwnerS
     slotName: "inputContainer",
 })(
     ({ ownerState }) => css`
+        overflow: hidden;
+        text-overflow: ellipsis;
```

Comment:
Is this something that should be changed for every field (because every field uses FieldContainer)? Couldn't this cause other problems?

Reply:
@juliawegmayr does this only affect `SelectField`/`FinalFormSelect`?
I would assume this is an issue that should be fixed in the theme of `MuiSelect` 🤔